gef-ui/styles/footer.scss

```diff
@@ -0,0 +1,3 @@
+.email-wrap {
+  word-break: break-all;
```

Comment:
I think `break-all` is OK for an email string, but only an email string. As it will only ever be a single string. All other text breaks should not use `break-all`, but `break-word` instead

Reply:
At the moment, all other words will have their letters break. For example, "Phone" could become like this on a screen:

Pho
ne

Reply:
done